docs/docs/getting-started/04-examples/04-jtc-example.md

```diff
@@ -0,0 +1,202 @@
+          load:
+            - controller: robot_state_broadcaster
+              hardware: hardware
+            - controller: joint_trajectory_controller
+              hardware: hardware
+    controllers:
+      robot_state_broadcaster:
+        plugin: aica_core_controllers/RobotStateBroadcaster
+        events:
+          transitions:
+            on_load:
+              switch_controllers:
+                hardware: hardware
+                activate: robot_state_broadcaster
+      joint_trajectory_controller:
+        plugin: aica_core_controllers/trajectory/JointTrajectoryController
+        events:
+          predicates:
+            has_trajectory_succeeded:
+              application: stop
+          transitions:
+            on_load:
+              switch_controllers:
+                hardware: hardware
+                activate: joint_trajectory_controller
+graph:
+  positions:
+    on_start:
+      x: 460
+      y: 0
+    stop:
+      x: 460
+      y: 660
+    buttons:
+      button:
+        x: 40
+        y: 840
+    hardware:
+      hardware:
+        x: 680
+        y: -20
+  buttons:
+    button:
+      display_name: Set desired trajectory
+      on_click:
+        call_service:
+          controller: joint_trajectory_controller
+          hardware: hardware
+          service: set_trajectory
+          payload: |-
+            {
+              frames: [start, waypoint_1, waypoint_2, waypoint_3, start], 
+              times_from_start: [2.0, 4.0, 6.0, 8.0, 10.0]
+            }
+  edges:
+    hardware_hardware_joint_trajectory_controller_has_trajectory_succeeded_on_stop_on_stop:
+      path:
+        - x: 440
+          y: 780
+        - x: 440
+          y: 700
+```
+</details>
+
+The application graph should show a hardware interface with a Joint Trajectory controller and an event trigger, as shown
+below:
+
+<div class="text--center">
+  <img src={jtcExampleOverview} alt="Simple JTC configuration" />
+</div>
+
+## The example explained
+
+The application starts by loading the Robot State broadcaster and Joint trajectory controllers for the generic six-axis
+robot. You may already play the application.
+
+After you press play, switch to the 3D view using the corresponding button on the top right of your AICA Studio. You
+should see your robot in its default joint configuration, along with 4 Cartesian frames named
+**start, waypoint_1, waypoint_2, and waypoint_3** that form a triangle. As you may have guessed, this application
+controls the robot such that it traverses the Cartesian frames in that order. 
+
+<div class="text--center">
+  <img src={jtcExampleFrames} alt="Cartesian frames for JTC" />
+</div>
+
+For now, let us go back to the graph view. Once there, turn your attention to the trigger button that connects to the
+`Set trajectory` service of the Joint Trajectory controller. Click on the gear icon on the edge that connects the button
+to the controller. You will see the following service payload string:
```

Comment:
```suggestion
to the controller. You will see the following service payload:
```
nitpick: seems overly technical

Reply:
you mean mentioning the (ROS) service like that right? not pointing out the payload